pex/resolver.py

```diff
@@ -151,7 +156,73 @@ def parsed_platform(platform=None):
   return Platform.create(platform) if platform and platform != 'current' else None
 
 
-class ResolveResult(namedtuple('ResolveResult', ['target', 'download_dir'])):
+class DownloadRequest(namedtuple('DownloadRequest', [
+  'targets',
```

Comment:
I guess that in this context `target` is a DistributionTarget? Unsure if worth spelling out, or less ambiguous in this codebase.

Reply:
Definitely unambiguous in this codebase - there are no other targets.